### ISSUE TITLE
Address not compliant with RFC7643

### ISSUE THREAD

**Issue by wwebers** [wso2/charon#180]
**Description:**
The address object not compliant with RFC7643 section 4.1.2 where it is defined as a multivalued complex datatype.

The current release (and probably even older ones) realizes Address just as complex datatype. As such the other mandatory attributes such as value or display are omitted.

**Affected Product Version:**
Current carbon release.

**OS, DB, other environment details and versions:**    
All OS
**Steps to reproduce:**
Take a look at [https://tools.ietf.org/html/rfc7643#section-4.1.2](https://tools.ietf.org/html/rfc7643#section-4.1.2)

**Comment by jrussellsmyth** [comment]
This is incorrect. The "addresses" attribute is multivalued, consisting of address complex objects.

addresses is defined as a multivalue object here
https://github.com/wso2/charon/blob/master/modules/charon-core/src/main/java/org/wso2/charon3/core/schema/SCIMSchemaDefinitions.java#L801-L811
- the fourth parmeter of createSCIMAttributeSchema is a boolean for "isMultivalue"

**Comment by wwebers** [comment]
Will have to check that again. However, I wrote the issue for 6 years ago and haven't followed the development since about 5 years as I never got any reply.
So, maybe this isn't an issue anymore, I don't know.
